packages/snfoundry/.env.example

```diff
@@ -1,10 +1,15 @@
-# ## Devnet
-# PRIVATE_KEY_DEVNET=0x57b2f8431c772e647712ae93cc616638
-# RPC_URL_DEVNET=http://127.0.0.1:5050
-# ## last staknet devnet account address
-# ACCOUNT_ADDRESS_DEVNET=0x4b3f4ba8c00a02b66142a4b1dd41a4dfab4f92650922a3280977b0f03c75ee1 
+## Devnet
+PRIVATE_KEY_DEVNET=0x71d7bb07b9a64f6f78ac4c816aff4da9
```

Comment:
Hi guys, don't touch the .env.example, restore the original please.

Reply:
@Akinbola247 lets keep the same ACCOUNT_ADDRESS_DEVNET, but the PRIVATE_KEY_DEVNET need to be updated since new version of devnet show the private key padded with a bunch of zeros